Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -18,6 +18,23 @@ extension XCTestCase {
     }
 }
 
+extension MongoClient {
+    internal func serverVersion() throws -> String {
+        let buildInfo = try self.db("admin").runCommand(["buildInfo": 1])
```

Comment:
Should this hard-code a primary read preference, just in case the client is using a different default read preference (e.g. `secondaryPreferred`)?